lib/pf/web/constants.pm

```diff
@@ -93,18 +93,15 @@ Readonly::Scalar our $URL_GAMING_REGISTRATION   => '/gaming-registration';
 Readonly::Scalar our $URL_DEVICE_REGISTRATION   => '/device-registration';
 Readonly::Scalar our $URL_DEVICE_REG_LOGOUT     => '/device-registration/logout';
 
-# External Captive Portal detection constant
-Readonly::Scalar our $REQ_CISCO                 => 'ap_mac';
-Readonly::Scalar our $REQ_MAC                   => 'mac';
-Readonly::Scalar our $REQ_ARUBA                 => 'apname';
-Readonly::Scalar our $REQ_CISCO_PORTAL          => '/cep(.*)';
```

Comment:
You want to handle requests coming in with the /cep* address ?
If yes, I'm not quite sure we should be backward compatible since it will involve adding other part of code for that kind of processing.
(Processing have been changed and is now using switch module used in the URI).
I can sure add an 'if' in the processing part to say if we match cep URI then use the Cisco module but we would have to mark it as deprecated and actually remove it before long...